src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ContentPresenter/ContentPresenter_NativeEmbedding.xaml

```diff
@@ -0,0 +1,47 @@
+<UserControl x:Class="Uno.UI.Samples.Content.UITests.ContentPresenter.ContentPresenter_NativeEmbedding"
+			 xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+			 xmlns:local="using:Uno.UI.Samples.Content.UITests.ContentPresenter"
+			 xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+			 xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+			 xmlns:not_skia="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:skia="http://uno.ui/skia"
+			 mc:Ignorable="d skia"
+			 xmlns:uc="using:Uno.UI.Samples.Converters"
+			 d:DesignHeight="300"
+			 d:DesignWidth="400">
+
+	<UserControl.Resources>
+		<uc:FromNullableBoolToVisibilityConverter x:Key="FromNullableBoolToVisibilityConverter"
```

Comment:
This is what I have on linux:

![image](https://github.com/unoplatform/uno/assets/5839577/23abacbf-df4f-491c-b137-406eac0c0235)

Reply:
Oh, I left `mpv` as the process to run in the sample code, but I should probably change it so that it tries a bunch of different options.